### PR TITLE
Use Content Loader for forms on the Create a DOS/G-Cloud 12 service

### PR DESCRIPTION
**Pull request by domoscargin**
https://trello.com/c/7s3UIoH3/987-3-replace-form-macros-with-dmcontentgovukfrontend-in-create-a-dos-service-journey

We want to use the content loader functions in `dmcontent.govuk_frontend` to generate our forms on this journey.

DOS and G-Cloud use the same files, so questions of type `checkbox_tree` need to special-cased until we've redesigned 'em. We use this type for selecting service categories on the G-Cloud journey

Another case popped up as I was working on this - we haven't yet accounted for multi-field pricing inputs, but there is an existing spike (https://github.com/alphagov/digitalmarketplace-content-loader/tree/ldeb-spike-pricing-input-multiple-fields), and I've created a ticket to implement this (https://trello.com/c/U3WM4YiT/617-2-add-multiple-fields-support-to-content-loader-pricing-input-questions).

### CODE REVIEW

**Review by bjgill — APPROVED** [review]
Code changes look reasonable. I assume you've tested locally that the output looks sensible? Any notable formatting changes?

**Comment by domoscargin** [comment]
> Code changes look reasonable. I assume you've tested locally that the output looks sensible? Any notable formatting changes?

I'm doing one final run-through of each of the journeys (4 G-Cloud and 3 DOS lots) before I merge, but I think the pricing input and checkbox_tree issues cover everything we haven't already accounted for.

**Comment by domoscargin** [comment]
Converting to draft while we wait on https://trello.com/c/7pW2lCrO/54-refactor-complex-supplier-questions

TLDR: There are multiple issues with conditional questions on this journey, and it's probably better to see if we can/should redesign the questions before this work.
